### DISCUSSION TITLE
How to solve the problem of single-machine memory explosion caused by large graphics?

### DISCUSSION THREAD

**Discussion by ghost** [alibaba/GraphScope#218] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by sighingnow** [comment, marked as answer]
Hi @illcat I have move the ticket to the discussion category.

GraphScope distributes large graphs to many fragments and those fragments can be placed on a set of machines in a large-scale cluster. The computation engines in GraphScope scales pretty well on distributed large graphs.
